ci/templates/ci/dashboard.html

```diff
@@ -180,6 +198,30 @@ <h2 class="dashhead-title">{{ dashboard.name }} Dashboard</h2>
       : [widget];
   }
 
+  function timeframeChange(selected) {
+    var start = '-1d';
+    switch (selected.value) {
+      case 'last-week':
+        start = '-7d';
+        break;
+      case 'last-30-days':
+        start = '-30d';
+        break;
+      case 'last-3-months':
+        start = '-90d';
+        break;
+      case 'last-year':
+        start = '-365d';
+        break;
+      case 'all-time':
+        pass
```

Comment:
This would make the value of `start` equal to `-1d`? Which I don't think would be correct..